java/gandiva/pom.xml

```diff
@@ -132,18 +132,15 @@
             <plugin>
                 <groupId>org.xolstice.maven.plugins</groupId>
                 <artifactId>protobuf-maven-plugin</artifactId>
-                <version>0.6.1</version>
-                <configuration>
-                    <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}
-                    </protocArtifact>
-                    <protoSourceRoot>proto</protoSourceRoot>
-                </configuration>
                 <executions>
+                    <id>src</id>
```

Comment:
This file is causing trouble in some of the CI pipelines

```
Error:  Malformed POM /arrow/java/gandiva/pom.xml: Unrecognised tag: 'id' (position: START_TAG seen ...<executions>\n                    <id>... @136:25)  @ /arrow/java/gandiva/pom.xml, line 136, column 25
 @ 
[INFO] 0 goals, 0 executed
Error:  The build could not read 1 project -> [Help 1]
Error:    
Error:    The project org.apache.arrow.gandiva:arrow-gandiva:16.0.0-SNAPSHOT (/arrow/java/gandiva/pom.xml) has 1 error
Error:      Malformed POM /arrow/java/gandiva/pom.xml: Unrecognised tag: 'id' (position: START_TAG seen ...<executions>\n                    <id>... @136:25)  @ /arrow/java/gandiva/pom.xml, line 136, column 25 -> [Help 2]
```

Reply:
Sorry, I forgot to enable the `arrow-jni` profile when building the project. Will address asap

Reply:
Pushed the fix